plugins/janus_sip.c

```diff
@@ -906,6 +906,13 @@ typedef struct ssip_oper_s ssip_oper_t;
 #undef NUA_HMAGIC_T
 #define NUA_HMAGIC_T	ssip_oper_t
 
+typedef struct janus_event_t
+{
```

Comment:
Code style: please put the opening bracket on the same line as the struct.
I noticed that in the functions/while/if blocks you added it's done too, so please update them accordingly.

Reply:
I'll do.